.github/workflows/python-package.yml

```diff
@@ -0,0 +1,35 @@
+name: Python Package
+
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Please replace all those steps with one step which prints `Hello World`. This is a placeholder for triggering the A-Team pipeline

Reply:
That makes more sense, ignore my previous comment :smiley:

Reply:
The uses step should be kept